src/overlays/misc/ovl_kaleido_scope/z_lmap_mark.c

```diff
@@ -27,9 +30,23 @@ void PauseMapMark_Init(PlayState* play) {
     gBossMarkState = 0;
     gBossMarkScale = 1.0f;
     gLoadedPauseMarkDataTable = gPauseMapMarkDataTable;
+#if PLATFORM_N64
+    if (B_80121AF0 != NULL) {
+        if (B_80121AF0->unk_34 != NULL) {
```

Comment:
Hmm I was going to suggest `(B_80121AF0 != NULL && B_80121AF0->unk_34 != NULL)` but we've been inconsistent on that